ui/components/app/collectible-default-image/collectible-default-image.js

```diff
@@ -9,14 +9,17 @@ export default function CollectibleDefaultImage({
   name,
   tokenId,
   handleImageClick,
+  handleImageKeyPress,
 }) {
   const t = useI18nContext();
   return (
     <div
+      tabIndex={0}
       className={classnames('collectible-default', {
         'collectible-default--clickable': handleImageClick,
       })}
       onClick={handleImageClick}
+      onKeyPress={handleImageKeyPress}
     >
```

Comment:
@georgewrmarshall , @darkwing : plz share your suggestion on above keyPress handler. I am not sure of it is ok to have this on div ?

Reply:
Hmm this is an interesting one. If I was going to look at it from a semantic html point of view in this context I believe the wrapping element should be an `<a/>` tag because it is navigating to another page or route in the application. The `react-router` `Link` component would be suitable but I see we don't use it in our codebase? Without complicating things too much I think `button` makes sense

Reply:
Actually I think conditional tag rendering might make sense here because there are use cases for it to be static. So something like 
```
  handleImageClick,
}) {
  const t = useI18nContext();
  const Tag = handleImageClick ? 'a' : 'div';
  return (
    <Tag
```

Reply:
This has been updated to use a `button` tag with conditional tag rendering